userProfile/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class UserprofileConfig(AppConfig):
```

Comment:
Please follow [pep 8](https://peps.python.org/pep-0008/#class-names) for naming convention. Also for the [app naming convention](https://learndjango.com/tutorials/django-best-practices-projects-vs-apps), maybe you can call it `user`?

Reply:
@natifridman Maybe it should be `users` because according to the name convention it should be the plural of the main model.

Reply:
That's correct